packages/core/core/src/factories/useFacetFactory.ts

```diff
@@ -16,7 +16,7 @@ const useFacetFactory = <SEARCH_DATA>(factoryParams: UseFacetFactoryParams<SEARC
     const error: Ref<UseFacetErrors> = sharedRef({}, `useFacet-error-${id}`);
 
     const search = async (params?: AgnosticFacetSearchParams) => {
-      Logger.debug('useFacet.search', params);
+      Logger.debug(`useFacet/${id}/search`, params);
```

Comment:
Look, in this factory `id` is optional. Maybe it would be okay to use `ssrKey` in this case

Reply:
Changed